tests/testthat/test-tbl-format.R

```diff
@@ -77,3 +77,10 @@ test_that("get_width_print()", {
   expect_equal(get_width_print(80), 80)
   expect_equal(get_width_print(140), 140)
 })
+
+test_that("format() signals an error if not all arguments in `...`are used", {
```

Comment:
Maybe add a motivating test case for passing down an argument to a `tbl_format_setup()` method?